libraries/AP_Mount/AP_Mount_Backend.h

```diff
@@ -319,8 +353,8 @@ class AP_Mount_Backend
     // structure for MAVLink Targeting angle and rate targets
     struct {
         MountTargetType target_type;// MAVLink targeting mode's current target type (e.g. angle or rate)
-        MountTarget angle_rad;      // angle target in radians
-        MountTarget rate_rads;      // rate target in rad/s
+        MountAngleTarget angle_rad;      // angle target in radians
```

Comment:
nitpick: let's line up the comments to appease my OCD

Reply:
Done